README.md

```diff
@@ -172,55 +172,62 @@ There is a breaking change on behavior expectations regarding speech and input h
-| [`23.a.hybrid-react-npm`](https://github.com/microsoft/BotFramework-WebChat/tree/master/samples/23.a.hybrid-react-npm)                                                                                                             | Demonstrates how to use different versions of React on a hosting app via NPM packages                                                                                                                                               | [Hybrid React Demo](https://microsoft.github.io/BotFramework-WebChat/23.a.hybrid-react-npm)                                                             |
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Sample&nbsp;Name&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Description                                                                                                                                                                                                                         | Link                                                                                                                                                           |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Migration**                                                                                                                                                                                                                      |                                                                                                                                                                                                                                     |                                                                                                                                                                |
```

Comment:
I know it's not ideal, but I intentionally did not add another column for directory groupings because it would severely limit the real estate available for listing the descriptions and sample names. If you have a better suggestion, please feel free. 
We can also iterate on this table after this major sample is merged in.